web/src/engine/osk/src/keyboard-layout/oskLayerGroup.ts

```diff
@@ -249,8 +259,42 @@ export default class OSKLayerGroup {
     return null;
   }
 
-  public refreshLayout(computedWidth: number, computedHeight: number) {
-    this.computedWidth = computedWidth;
-    this.computedHeight = computedHeight;
+  public resetPrecalcFontSizes() {
```

Comment:
Note:  should also reset ._heightPadding, as it exists to cache a stylesheet value and is only available at the same time.